data/hooks/diagnosis/50-systemresources.py

```diff
@@ -103,6 +103,26 @@ def run(self):
 
             yield item
 
+        #
+        # Check for minimal space on / + /var
+        # because some stupid VPS provider only configure a stupidly
+        # low amount of disk space for the root partition
+        # which later causes issue when it gets full...
+        #
+
+        main_disk_partitions = [d for d in disk_partitions if d.mountpoint in ['/', '/var']]
+        main_space = sum([psutil.disk_usage(d.mountpoint).total for d in main_disk_partitions])
+        if main_space < 10 * GB:
+            yield dict(meta={"test": "rootfstotalspace"},
+                       data={"space": human_size(main_space)},
+                       status="ERROR",
+                       summary="diagnosis_rootfstotalspace_critical")
+        if main_space < 14 * GB:
```

Comment:
In messages you talk about 16GB but here you put 14, is it intended?

Reply:
hmyeah idk that's pretty much arbitrary number, the difference is more like a margin to not have a warning if you have like 15.5 GB ... 16 GB recommended, 15 GB maybe ok, but if you have 14 GB this is definitely an issue

Reply:
Seems legit!